fili-wikipedia-example/Dockerfile

```diff
@@ -26,10 +26,7 @@ RUN mvn -U -B install -DskipTests=true -Dmaven.javadoc.skip=true -Dcheckstyle.sk
 
 EXPOSE 9998
 
-ENTRYPOINT export HOSTIP="$(resolveip -s $HOSTNAME)" \
-&& exec /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf \
- & mvn -pl fili-generic-example exec:java -Dbard__fili_port=9998 \
-    -Dbard__druid_coord=http://localhost:8081/druid/coordinator/v1 \
-    -Dbard__non_ui_druid_broker=http://localhost:8082/druid/v2 \
-    -Dbard__ui_druid_broker=http://localhost:8082/druid/v2
+COPY entrypoint.sh /entrypoint.sh
```

Comment:
I updated the Dockerfile and ```entrypoint.sh``` and uploaded the new image to Dockerhub to test. It seems to work on my computer, but then I tried pulling and running the image from a different computer and got this message when I sent the query:
```
{
    "status" : 400,
    "statusName" : "Bad Request", 
    "reason" : "com.yahoo.bard.webservice.web.BadApiRequestException",
    "description" : "Table name 'wikipedia' does not exist.",
    "druidQuery" : null 
}
``` 
I'm not sure what's wrong.